### PR TITLE
fix: adjust images' dimension

### LINKED ISSUE #158: adjust images' dimention
image

**After upgrading to V2, Here is some CSS that needs to be changed：**
1. The default font color of the navigation is `#fff`
2. the image width needs to be deleted~

Thanks~

### PR DESCRIPTION
**Pull request by KishaniKandasamy**
Fixes: #158

Changes:
I've updated customTheme.css

### CODE REVIEW

**Comment by juzhiyuan** [comment]
@guoqqqi Please review when you have time.

File: website/src/css/customTheme.css [hunk 1]
```diff
@@ -8,10 +8,6 @@
   --ifm-color-primary-darker: #9a2521;
   --ifm-color-shadow--lw: #7f1e1b;
 }
-.navbar {
-  background: var(--ifm-color-primary);
-  color: var(--ifm-color-primary-lightest);
-}
```
Comment (guoqqqi) [inline website/src/css/customTheme.css:14]: we would prefer the text color of the navigation bar to be #fff, like this

```css
.navbar a {
  color: #fff;
}
```

Comment (guoqqqi) [reply website/src/css/customTheme.css:14]: It is my fault for not expressing clearly, thank you for your help～

Comment (KishaniKandasamy) [reply website/src/css/customTheme.css:14]: > we would prefer the text color of the navigation bar to be #fff, like this
> 
> ```css
> .navbar a {
>   color: #fff;
> }
> ```

Yesss but here  nav bar style was already set to primary style(in docusaurus.config.js) so I thought we could remove  these lines from  css :)
.navbar {
  background: var(--ifm-color-primary);
  color: var(--ifm-color-primary-lightest);
}

Comment (KishaniKandasamy) [reply website/src/css/customTheme.css:14]: When we change the default  text color of the navigation bar to be #fff ..we also need to add hover and active selector to it IMHO :)
Screenshot (67)

Comment (juzhiyuan) [reply website/src/css/customTheme.css:14]: Yep, we'd better support Hover haha

**Comment by LiteSun** [comment]
any update? 😊

**Comment by juzhiyuan** [comment]
Hi, @KishaniKandasamy, I just fixed the auto deployed issue, we could continue working on this one :)

**Pull request closed without merging**